### DISCUSSION TITLE
Unverified email prevents legitimate  email to be registered

### DISCUSSION THREAD

**Discussion by guitcastro** [keycloak/keycloak#13760] [Feedback]
If a user which does not own the email try to register it, even if not validated is enough to prevent the legitimate email owner to use the email. 

Enabling duplicated emails will force to disable login with email, which is not a viable solution. 

Is this the desired behavior?  There is any workaround to avoid this issue?

**Comment by lexcao** [comment]
> If the legitimate email owner wants to use the email but it is already registered

the user could try `forgot-password` by email then he could own the account.
